app/templates/frameworks/reuse_declaration.html

```diff
@@ -31,21 +29,43 @@
 {% block mainContent %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-l">{{ self.page_title_inner() }}</h1>
+      {% set question_advice %}
+        <p class="govuk-body">In {{ old_framework_application_close_date|nbsp }}, your organisation completed a declaration for {{ old_framework.name }}.</p>
+        <p class="govuk-body">You can reuse some of the answers from that declaration.</p>
+        <p class="govuk-body">You’ll need to:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>review the answers you gave before and make sure they’re still correct</li>
+          <li>provide some new answers for this declaration</li>
+        </ul>
+      {% endset %}
 
-      <p class="govuk-body">In {{ old_framework_application_close_date|nbsp }}, your organisation completed a declaration for {{ old_framework.name }}.</p>
-      <p class="govuk-body">You can reuse some of the answers from that declaration.</p>
-      <p class="govuk-body">You’ll need to:</p>
-      <ul class="govuk-list govuk-list--bullet">
-        <li>review the answers you gave before and make sure they’re still correct</li>
-        <li>provide some new answers for this declaration</li>
-      </ul>
-
-      <form method="POST" action="{{ url_for(".reuse_framework_supplier_declaration_post", framework_slug=current_framework.slug) }}">
+      <form method="POST" action="{{ url_for('.reuse_framework_supplier_declaration_post', framework_slug=current_framework.slug) }}">
         <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
         <input type="hidden" name="old_framework_slug" value="{{ old_framework.slug }}"/>
 
-        {{ form.reuse }}
+        {{ govukRadios({
+          "classes": "govuk-radios--inline",
+          "idPrefix": "input-" + form.reuse.name,
+          "name": form.reuse.name,
+          "hint": {
+            "html": question_advice,
+            "classes": "dm-question-advice",
```

Comment:
Are we using this anywhere but on Briefs Responses to override the default grey colouring? https://github.com/search?q=dm-question-advice&type=code

Reply:
(ie: does it do anything here?)

Reply:
Curious. briefs-frontend has [the override](https://github.com/alphagov/digitalmarketplace-brief-responses-frontend/blob/d9fe9616f598e33fd5e86d28bc47e6664ce77985/app/assets/scss/overrides/_hint.scss), but not supplier-frontend.

You're quite right. It has no effect. I removed line 52 and nothing changed. I'll remove it, since it seems a bit pointless.

Reply:
🤔 so why isn't it grey?

Reply:
I think Description and Question Advice fields get parsed into HTML with govuk classes. So the elements within a complex formatted hint have things like `govuk-body` which set the text colour and override the grey hint text.

It's not ideal.

Reply:
I seee.... confusion abounds

Reply:
Especially since right now hints are contained within a `<span>`, so having block level elements inside that is technically invalid HTML.

govuk-frontend v3 changes this to a `<div>`

Reply:
😬